development/otp-input-app/pages/otp-input/index.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: "ReuseJS otp-input"
+---
+
+import Sample from "../../components/otp-input/Sample/Sample.tsx"
+import ReuseExample from "../../components/otp-input/ReuseExample/ReuseExample.tsx";
+import ReuseNumberExample from "../../components/otp-input/ReuseNumberExample/ReuseNumberExample.tsx";
+import ReuseNumberWithPasteExample from "../../components/otp-input/ReuseNumberWithPasteExample/ReuseNumberWithPasteExample.tsx";
+
+
+## Table of contents
+
+## List of exported Components:
+
+### HeadlessOtpInput 
+
+This is the OTP input component with no inherit styles. user will have to provide styles to the component.
+
+This OTP input component we can customize the number of inputs based on requirements we can pass the prop `numInputs` with value.
+
+We need to use the HTML input element to render the boxes. and if we need to add wrapper to input components with help of renderInput prop. and `InputProps` It allows the basic HTML input element allowed props.
```

Comment:
You can use ChatGPT or other tools like grammarly to check spelling and make the docs more readable @Subbu-betalectic